CHANGELOG.md

```diff
@@ -2,13 +2,37 @@
 
 # Master
 
-- [ENHANCEMENT] Add option `--insecure-proxy` to `serve` to allow for proxying
-  self signed SSL certificates.
-
 ### Applications
 ### Addons
 ### Blueprints
 
+### 0.1.7
+
+- Users
+  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
+  + Replace `ember-cli-esnext` with `ember-cli-6to5`:
```

Comment:
we should also tell people that if the module transpile step is stil es3 safe. Aspects of 6to5 are fine on es3, but if they are concerned about the temptation they can remove it package.jso entirely..